cpp/src/barretenberg/env/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(env)
```

Comment:
Without including the ecc module I would get undefined reference errors for `bbmalloc` inside the `env_load_prover_crs` and `env_load_verifier_crs` methods. I was originally able to work around this by using `aligned_alloc` inside those methods (`aligned_alloc` is just the internal method to `bbmalloc`). I could try going back to that.

The usage of the env crs came about because @zac-williamson realized it could be used instead of having to store the reference string from the native verification key in the stdlib verification key. This was before the solidity tests were added though. I reverted back to storing the reference string as I kept getting errors when trying to use the env crs with the solidity helpers. So theoretically I could revert this module change and remove all usage of the env crs in the stdlib recursive verifier tests, but it still leaves this issue with the env crs (I will make an issue on the repo if we go with this strategy).

Perhaps there is someone more familiar with cmake I can request takes a look at this?

Reply:
I have removed this so it is no longer an issue